test/FunctionalTests/Microsoft.OData.Client.Tests/metadata/ClientTypeUtilTests.cs

```diff
@@ -38,11 +38,12 @@ public void IFTypeProperty_HasNoKeyAttribute_TypeIsNotEntity()
             //Assert
             Assert.Equal(expectedResult, actualResult);
         }
+#endif
 
     }
     public class Person
     {
-        [m.Key]
+        [System.ComponentModel.DataAnnotations.Key]

```

Comment:
This annotation is only available for ```#if !PORTABLELIB``` so the whole class probably needs to be wrapped in the ```#if```